hw-7/7.2/style.css

```diff
@@ -0,0 +1,139 @@
+ * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Nunito', sans-serif;
+}
+
+body {
+    max-height: 100%;
+    background-color:  #3399CC;
+}
+
+h1 {
+    margin-bottom: 40px;
+}
+
+.wrapper {
+    width: 65%;
+    position: absolute;           
+    top: 50%;  
+    left: 50%;                       
+    transform: translate(-50%, -50%);
+    color: #fff;    
+}
+
+ul {
+    border: 2.5px solid #fff;
+    font-size: 0;
+    height: 60vh;
+    margin-bottom: 30px;
+}
+
+ul li {
+    list-style: none;
+    display: inline-block;
+    font-size: 14px;
+    text-align: center;
+    text-transform: capitalize;
+    vertical-align: middle;
+    border: 2.5px solid #fff;
+    position: relative;
+}
+
+ul:hover li {
+    opacity: 0.5;
+}
+
+ul:hover li:hover {
```

Comment:
I believe you should  rely only for hover on li element,
Overall, good job for that task too